### PR TITLE
Add issuer to message context when JWT.

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
Fix https://github.com/wso2/product-apim/issues/9181

Add the issuer claim of the JWT to the message context upon a successful JWT validation.

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityUtils.java [hunk 1]
```diff
@@ -45,6 +45,9 @@ public static void setAuthenticationContext(MessageContext synCtx,
                                                 String contextHeader) {
         synCtx.setProperty(API_AUTH_CONTEXT, authContext);
         synCtx.setProperty(APIConstants.API_KEY_TYPE, authContext.getKeyType());
+        if (authContext.getIssuer() != null) {
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityUtils.java:48]: can authContext ever be null?

Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityUtils.java:48]: It's "authContext.getIssuer()" issuer is null checked here.

Comment (isharac) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityUtils.java:48]: What if authContext is null?

Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/security/APISecurityUtils.java:48]: Checked all occurrences of method calls and params in the current code base; authcontext is not null.

**Review by isharac — APPROVED** [review]

**Pull request merged**